UPGRADE.asciidoc

```diff
@@ -159,6 +159,13 @@ Add default values to new auth source parameters
 
     /usr/local/pf/addons/upgrade/to-7.1-authentication-conf.pl
 
+Fix the Ubiquiti typo
+^^^^^^^^^^^^^^^^^^^^^
+
+In order to use the new Ubiquiti switch module renamed, run the following command:
```

Comment:
the new Ubiquiti switch module renamed -> the Ubiquiti switch module that has been renamed